src/database/seeders/ProjectSettingSectionsWithItemsSeeder.php

```diff
@@ -57,6 +57,10 @@ protected static function addSectionSettings(Collection $allProjectSettingTypes,
 
             if (!$settingType) continue;
 
+            $existingSetting = $projectSettingSection->projectSettings()->where('key', $projectSettingData['key'])->first();
```

Comment:
**Blocking**

need to get all existing keys once then compare the keys inside foreach with it in order to make only one query not a query for each key.

in other words we may have the collection of all existing keys in database and run where() method on it instead of running it on a relationship method